Helper/Aperture.php

```diff
@@ -165,7 +167,7 @@ public function register($firstName, $lastName, $company, $phone, $email, $passw
      *
      * @return array() $response
      */
-    public function executeAuthRequest ($service_url, $method = 'GET', $payload = null) {
+    public function executeAuthRequest ($service_url, $method = 'GET', $payload = null, $timeout = self::DEFAULT_CURL_TIMEOUT) {
```

Comment:
would prefer if variables like this included the units, eg `$timeout_seconds` and `self::DEFAULT_CURL_TIMEOUT_SECONDS` to avoid hunting for docs each time.